### PR TITLE
Add a chemiscope example.

### PR DESCRIPTION
**Pull request by andrewtarzia**
Just adding a small example to raise awareness of the utility of the tool. I am happy to change the scope as necessary.

Note that I have not made the python code testable to avoid the need for a new dev dependancy,

### CODE REVIEW

**Comment by lukasturcani** [comment]
i think having the code testable is worth a dev dependcy

**Comment by andrewtarzia** [comment]
Added the dependency, tests should pass.

**Review by lukasturcani — APPROVED** [review]
happy to merge after backticks are fixed

File: docs/source/basic_examples.rst [hunk 1]
```diff
@@ -1130,6 +1130,109 @@ from the database. For example,
...
+---------------
+
+:mod:`chemiscope` can be used through the `web-interface`__ or can be embedded
+into read-the-docs pages through a sphinx package, like for `toy models`__.
+
+__ https://chemiscope.org/
+
+__ https://cgmodels.readthedocs.io/en/latest/cg_model_jul2023_2p3_ton.html
+
+Either way, you need to write a `.json` or `.json.gz` file
```
[the remark this answers is not in the document]
Comment (andrewtarzia) [reply docs/source/basic_examples.rst]: Fixed, ty!

**Pull request merged**